thepeg.sh

```diff
@@ -1,11 +1,11 @@
 package: ThePEG
 version: "%(tag_basename)s"
-tag: "v2.2.2-alice2"
+tag: "v2.2.2-alice3"
```

Comment:
Please revert to alice2. I had already merged a PR upgrading ThePEG to the latest tag.